website/snapshots.yml

```diff
@@ -0,0 +1,31 @@
+- name: Home page dark theme
+  url: https://wix.github.io/Detox/
+  execute: document.querySelector('div.toggle_vylO.colorModeToggle_DEke > button').click('data-theme', 'dark')
```

Comment:
There are three issues about this approach:

1. Button click function does not accept parameters. What you did was just a wild guess, and, fortunately for you, it was redundant (hence, ignored) – so that you did not notice that something was wrong.
2. `div.toggle_vylO.colorModeToggle_DEke` CSS selector is unreasonably specific. [Check out what is a specificity of a CSS selector.](https://specificity.keegan.st)
3. Even if you used only the really meaningful parts in that CSS selector, e.g.: `.toggle_vylO.colorModeToggle_DEke > button`, your problem is that `cssClasses_g1BBerIsh` are randomly suffixed, so the chances are that your click won't work after first change to website code or after updating a Docusaurus version.

Reply:
I again suggest to you to consider using an CSS attribute selector over the `aria-label` of that toggle button. Please re-read: https://www.w3schools.com/css/css_attribute_selectors.asp

Reply:
@noomorph Thanks for the details regarding this approach. I have reviewed all and re-read the docs to understand better. 
Yes, instead of using a CSS selector, it would be better to use an attribute selector based on the aria-label of the toggle button. This would make the selector more robust and less likely to be affected by changes in the website's code. 

Decided to modify the approach to select the html attribute 'data-theme' to target dark and light modes due to a bug in Docusaurus on the toggle button.